.travis.yml

```diff
@@ -1,10 +1,22 @@
 language: node_js
 
-node_js:
-  - "6"
-  - "7"
-  - "8"
-  - "stable"
+matrix:
+  include:
+    - os: linux
+      node_js: "stable"
+      env: JOB_PART=lint
+    - os: linux
+      node_js: "stable"
+      env: JOB_PART=integration
+    - os: linux
+      node_js: "8"
+      env: JOB_PART=integration
+    - os: linux
+      node_js: "7"
```

Comment:
7 is a "in development" version number. Only even version numbers are node.js version you need to test.

Reply:
The the versions will be stable, 8 and 6? As far as I understood webpack 4 won't support `v4` anymore.
Also, are the labels required for webpack-bot? I don't see it doing anything on new PRs.